lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -193,8 +193,11 @@ uint32_t SessionState::get_current_version() {
   return current_version_;
 }
 
-void SessionState::set_current_version(int new_session_version) {
+void SessionState::set_current_version(int new_session_version,
+	           SessionStateUpdateCriteria& uc) {
```

Comment:
`uc` -> `session_uc` please. :) We are trying to be more consistent with naming parameters.

Reply:
Adapted